build.gradle

```diff
@@ -18,46 +24,85 @@ repositories {
     jcenter()
 }
 
+publishing {
+    publications {
+        customLibrary(MavenPublication) {
+            from components.java
+        }
+    }
+
+    repositories {
+        maven {
+            name = 'mt.edu.um.cf2.jgribx'
+            url = layout.buildDirectory.dir("repo")
+        }
+    }
+}
+
 sourceSets {
-	main {
-		resources.srcDirs = ["src/main/java","src/main/resources"]
-	}
-	test {
-		resources.srcDirs = ["src/test/java", "src/test/resources"]
-	}
+    main {
+        resources.srcDirs = ["src/main/java", "src/main/resources"]
+    }
+    test {
+        resources.srcDirs = ["src/test/java", "src/test/resources"]
+    }
 }
```

Comment:
Out of curiosity, can this block be removed and everything still function correctly at runtime? 

The default main resource directory is `src/main/resources` (which is redeclared here) and the same goes for the test resource directory (`src/test/resources`), and I don't seen any resources in the respective `src/*/java` directories so we should be okay to remove all of this.